packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.html

```diff
@@ -0,0 +1,114 @@
+<div class="telecom-pack-xdsl-meetings">
+    <header class="widget-presentation-header">
+        <h2
+            class="widget-presentation-title"
+            data-translate="xdsl_meeting_slots"
+        ></h2>
+    </header>
+
+    <!-- LOADER -->
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.errorMessage">
+        <span data-ng-bind-html="$ctrl.errorMessage"></span>
+    </oui-message>
+
+    <oui-message type="success" data-ng-if="$ctrl.successMessage">
+        <span data-ng-bind-html="$ctrl.successMessage"></span>
+    </oui-message>
+
+    <div
+        class="widget-presentation py-3 move-number-config ft-meeting"
+        data-ng-if="$ctrl.showMeetingSlots && !ctrl.loading && !$ctrl.successMessage"
+    >
+        <div class="row">
+            <div
```

Comment:
**suggestion:** `div.font-weight-bold` can be turned into a `<strong>` or it can be turned into an `<h3>` element to act as a title for `.widget-presentation` element.

Is that make sense for you?